lib/micro_cms/engine.rb

```diff
@@ -19,7 +20,7 @@ class Engine < ::Rails::Engine
 
     initializer 'micro_cms.include_view_helpers' do |_app|
       ActiveSupport.on_load :action_view do
-        ActionView::Base.public_send :include, MicroCms::CmsBlockHelper
+        include MicroCms::CmsBlockHelper
```

Comment:
Do these changes still work on Rails 6 projects?

Reply:
Yes, I tested it on Schuler 👍🏼